src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/CustomerRegisterSuccess.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+class Shopgate_Cloudapi_Model_Frontend_Observer_CustomerRegisterSuccess
+{
+    /**
+     * Checks if the order is received from Shopgate API call.
+     *
+     * @param Varien_Event_Observer $observer
+     */
+    public function execute(Varien_Event_Observer $observer)
+    {
+        /* @todo-sg - Check is shopgate App register */
```

Comment:
one more thing about this one, please make sure to return before the redirect. Something like an early return right after the ```todo```. Why this is important as this observer currently breaks normal create customer functionality. We want to deploy safe code in case we forget to fix this todo.

Reply:
done